lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -817,8 +817,7 @@ llvm::json::Value CreateStackFrame(lldb::SBFrame &frame) {
     if (line && line != LLDB_INVALID_LINE_NUMBER)
       object.try_emplace("line", line);
```

Comment:
From the [DAP specification](https://microsoft.github.io/debug-adapter-protocol/specification#Types_StackFrame) it looks like `line` is also not optional. Can you make sure that `line` is always added as well?

Note that `LLDB_INVALID_COLUMN_NUMBER` is 0 so we don't need to worry about that one. The same is not true for `LLDB_INVALID_LINE_NUMBER` which has the value of `UINT32_MAX`. You'll need to do some checking since the specification says that the line number should be 0 when the line should be ignored by the client.